components/vapi/actions/update-assistant-settings/update-assistant-settings.mjs

```diff
@@ -0,0 +1,263 @@
+      optional: true,
+    },
+    voice: {
+      type: "object",
+      label: "Voice",
+      description: "A formated JSON object  for the assistant's voice. **Example: {\"provider\":\"tavus\",\"voiceId\":\"r52da2535a\",\"callbackUrl\":\"voice\",\"chunkPlan\":{\"enabled\":true,\"minCharacters\":30,\"punctuationBoundaries\":[\"。\",\"，\",\".\",\"!\",\"?\",\";\",\"،\",\"۔\",\"।\",\"॥\",\"|\",\"||\",\",\",\":\"],\"formatPlan\":{\"enabled\":true,\"numberToDigitsCutoff\":2025}},\"conversationName\":\"voice\",\"conversationalContext\":\"voice\",\"customGreeting\":\"voice\",\"fallbackPlan\":{\"voices\":[{\"provider\":\"tavus\",\"voiceId\":\"r52da2535a\"}]},\"personaId\":\"voice\",\"properties\":{\"maxCallDuration\":1.1,\"participantLeftTimeout\":1.1,\"participantAbsentTimeout\":1.1,\"enableRecording\":true,\"enableTranscription\":true,\"applyGreenscreen\":true,\"language\":\"language\",\"recordingS3BucketName\":\"recordingS3BucketName\",\"recordingS3BucketRegion\":\"recordingS3BucketRegion\",\"awsAssumeRoleArn\":\"awsAssumeRoleArn\"}}**. [See the documentation](https://docs.vapi.ai/api-reference/assistants/update) for futher details",
+      optional: true,
+    },
+    firstMessage: {
+      type: "string",
+      label: "First Message",
+      description: "The first message the assistant will say or a URL to an audio file. If unspecified, assistant will wait for user to speak and use the model to respond once they speak.",
+      optional: true,
+    },
+    firstMessageMode: {
+      type: "string",
+      label: "First Message Mode",
+      description: "Mode for the first message",
+      optional: true,
+      options: FIRST_MESSAGE_MODE_OPTIONS,
+    },
+    hipaaEnabled: {
+      type: "boolean",
+      label: "HIPAA Enabled",
+      description: "When this is enabled, no logs, recordings, or transcriptions will be stored. At the end of the call, you will still receive an end-of-call-report message to store on your server.",
+      optional: true,
+    },
+    clientMessages: {
+      type: "string[]",
+      label: "Client Messages",
+      description: "These are the messages that will be sent to your Client SDKs",
+      options: CLIENT_MESSAGE_OPTIONS,
+      optional: true,
+    },
+    serverMessages: {
+      type: "string[]",
+      label: "Server Messages",
+      description: "These are the messages that will be sent to your Server URL",
+      options: SERVER_MESSAGE_OPTIONS,
+      optional: true,
+    },
+    silenceTimeoutSeconds: {
+      type: "integer",
+      label: "Silence Timeout Seconds",
+      description: "How many seconds of silence to wait before ending the call. Defaults to 30",
```

Comment:
Should probably have the 'default' property set instead of putting it in the description